packages/ui/src/components/EventsCard/EventsCard.mocks.ts

```diff
@@ -0,0 +1,33 @@
+export const MAIN_TITLE = 'Najbliższe terminy';
+
+export const BUTTON_TEXT = 'Przejdź do wydarzenia';
```

Comment:
🤔 Those constants aren't mocks and probably should be located in the `EventsCard.tsx` file